app/components/attachment_button/index.js

```diff
@@ -498,6 +499,7 @@ export default class AttachmentButton extends PureComponent {
 
         return (
             <TouchableWithFeedback
+                {...accessibilityProps('attachment button')}
```

Comment:
Looks like the original accessibility labels I provided are still used, e.g. ends with button/input . Should these be cleaned up as well or should we wait after the discussion with accessibility experts?

Reply:
`AccessibilityLabel` should be localized

Reply:
Now localized. Accessibility labels are now consistent, ending with button, input, etc.

Reply:
- ``const accessibilityLabel = {...}`` are consistently added at the bottom of each file.
- react-intl via context
- translation string has prefix of ``accessibility.``